sonar-text-plugin/src/test/java/org/sonar/plugins/secrets/SecretsRulesDefinitionTest.java

```diff
@@ -49,12 +49,11 @@ void define_rules() {
     assertThat(repository.rules()).hasSize(SecretsRulesDefinition.checks().size());
     assertThat(repository.name()).isEqualTo("Sonar Secrets Analyzer");
 
-    RulesDefinition.Rule ruleS7529 = repository.rule("S6290");
-    assertThat(ruleS7529).isNotNull();
-    assertThat(ruleS7529.name()).isEqualTo("Amazon Web Services credentials should not be disclosed");
-    assertThat(ruleS7529.activatedByDefault()).isTrue();
-    assertThat(ruleS7529.htmlDescription()).contains("AWS credentials are designed to authenticate and authorize requests to AWS.");
```

Comment:
As the description of rule S6290 changed, this test failed. As I don't see any value of this line in particular, I removed it.